### ISSUE TITLE
[Security Issue] Define Cog Displays API Key

### ISSUE THREAD

**Issue by Exper1mental** [ClemsonCPSC-Discord/ClemBot#241]
Under special circumstances, found by @CodysGithub, the `define` cog will report the API key in use. This could be used maliciously to flood the key with API requests, making it unusable or causing it to be revoked.

`!define ../../../../../robots.txt` will do this, yielding the following:

```
ERROR: Command exception
Exception:
Command raised an exception: ContentTypeError: 0, message='Attempt to decode JSON with unexpected mimetype: text/plain; charset=utf-8', url=URL('https://www.dictionaryapi.com/robots.txt?key=**KEY REMOVED**)]
```

It seems to me that the easiest solution would be some form of error catch that checks the error description of the error for whether the API key is present. If it is, the key is removed, and then the error is displayed as it normally would.

This issue is also present in my weather cog right now, so I'll need to fix that.

**Comment by Jay-Madden** [comment]
ohh very interesting, what an obscure find well done. We should def fix this asap

cc: @RajatSethi2001

**Comment by Exper1mental** [comment]
Fixed per #242
